app/javascript/react/containers/ShopIndex.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react'
+import ShopTile from '../components/ShopTile'
+
+class ShopIndex extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      shops: []
+    }
+  }
+
+  componentDidMount(){
+
+    fetch(`/api/v1/shops`)
+    .then(response => {
+      if (response.ok) {
+        return response;
+      } else {
+        let errorMessage = `${response.status} (${response.statusText})`,
+        error = new Error(errorMessage);
+        throw(error);
+      }
+    })
+    .then(response => response.json())
+    .then(body => {
+      console.log(body)
+      this.setState({ shops: body.shops})
+    })
+    .catch(error => console.error(`Error in fetch: ${error.message}`));
+  }
+
+
+  render() {
+
+
+    let mappedShops = this.state.shops.map(shop => {
+      return(
+        <div key={shop.id} className='shop-tile'>
+        <ShopTile
+          key = {shop.id}
+          id = {shop.id}
+          name = {shop.name}
+          address = {shop.address}
+          city = {shop.city}
+          state = {shop.state}
+          phone = {shop.phone}
+          url = {shop.url}
+        />
+        </div>
```

Comment:
The `div` surrounding this shopTile should go _inside_ the shopTile component!

Reply:
done and done.